samples/DatabaseProjectAFSqlExtensions/DatabaseProjectAFSqlExtensions.sqlproj

```diff
@@ -0,0 +1,86 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
I'd suggest just putting this in https://github.com/Azure/azure-functions-sql-extension/tree/main/samples/Database and using the scripts there. That way we don't have to remember to go update these ones as well when we add/modify anything.

Reply:
When I update the PR with the SDK-style, you will see that I named the folder differently. When it gets committed, I think I will suggest removing the prior sql scripts.